sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/query.sql

```diff
@@ -307,14 +351,16 @@ _previous AS (
     `moz-fx-data-shared-prod.telemetry_derived.clients_first_seen_v2`
 )
 SELECT
-  {% if is_init() %}
-    IF(_previous.client_id IS NULL, _current, _previous).*
-  {% else %}
-    -- For the daily update:
-    -- The reported ping status in the metadata is updated when it's NULL.
-    -- The second_seen_date is updated when it's NULL and only if there is a
-    -- main ping reported on the submission_date.
-    -- Every other attribute remains as reported on the first_seen_date.
+{% if is_init() %}
```

Comment:
For initialization there is no need to compare with the existing records. Is it possible to have a client_id with different sample_id that would create a duplicate due to inserting directly to the table and in parallel?

Reply:
No, `(client_id, sample_id)` pairs should be stable

Reply:
Thanks. Probably worth a data check, for when we implement them for this table.